lib/validate.js

```diff
@@ -81,10 +81,27 @@ function validateRendition(rendition, index, location){
             }
         }
     }
+    if (rendition.watermark && rendition.watermark.watermarkContent !== undefined && rendition.watermark.watermarkContent !== null) {
+        validateWatermark(rendition.watermark);
+    }
 }
 
+function validateWatermark (paramsWatermark) {
+    // Only validates that content is an HTTPS or data URL
+    if (!process.env.WORKER_TEST_MODE) {
+
+        if (paramsWatermark.watermarkContent.startsWith('data:')) {
+            if (paramsWatermark.watermarkContent.startsWith('data:image/png') || !validDataUrl(paramsWatermark.watermarkContent)) {
```

Comment:
The condition is wrong. `image/png` is what we want.
Either the first part should be `!paramsWatermark.watermarkContent.startsWith('data:image/png')` or it should be `&&` instead of `||`. Not 100% sure what you want to achieve here.

Reply:
+1, also add a test case covering this

Reply:
yup correct `not data:image/png or inValidDataUrl` then throw error
 `!paramsWatermark.watermarkContent.startsWith('data:image/png')` seems appropriate here. will edit the tests too.